docs/hour-of-code/educators-2021.html

```diff
@@ -398,12 +398,93 @@ <h5 class="display-font">Assign in Teams</h5>
                     </div>
                 </div>
             </section>
-
+            <section class="hoc-section">
+                <h3 class="display-font title sunflower">
+                    Free Trainings and Live Lessons
+                </h3>
+                <p>
+                  Further prepare with educator trainings that will cover running
```

Comment:
Again "trainings" should be singular, and typo with "Gam". 
I might reword this sentence, "Join free educator training sessions that will prepare you to facilitate the Hour of Code activity in your classroom, and participate in the Global MakeCode Arcade Game Jam."